### ISSUE TITLE
Object doesn't support property or method 'assign', on Internet Explorer

### ISSUE THREAD

**Issue by actual-saurabh** [gocodebox/lifterlms#1019]
### Reproduction Steps

No special steps needed. On IE, this line produces an error
https://github.com/gocodebox/lifterlms/blob/master/assets/js/llms.js#L1902 because this method is unsupported in IE: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Browser_compatibility

This is ideally resolved by using a modern browser which can be Edge as promoted by Microsoft.

### Expected Behavior

NA

### Actual Behavior

NA

### Error Messages / Logs

```
SCRIPT438: Object doesn't support property or method 'assign'
llms.min.js (1,15622)

```

### System Information

NA

### Browser, Device, and Operating System Information

+ IE (all versions)
+ Windows 7+

**Comment by thomasplevy** [comment]
Source file: https://github.com/gocodebox/lifterlms/blob/master/assets/js/app/llms-tracking.js#L99

**Comment by thomasplevy** [comment]
Since WP core support ie >= 11 we should as well: https://make.wordpress.org/core/handbook/best-practices/browser-support/

**Comment by thomasplevy** [comment]
We can replace assign with jQuery `$.extend` without adding any new dependencies

**Comment by thomasplevy** [comment]
@actual-saurabh & @eri-trabiccolo this is resolved in 3.37.8 which will be released in the next few hours. Please update support tickets accordingly.

**Comment by thomasplevy** [comment]
@actual-saurabh & @eri-trabiccolo correction: 3.37.9
